tests/platform_tests/api/test_fan_drawer.py

```diff
@@ -49,6 +55,8 @@ def test_get_name(self, duthost, localhost, platform_api_conn):
 
             if self.expect(name is not None, "Unable to retrieve Fan_drawer {} name".format(i)):
                 self.expect(isinstance(name, str), "Fan_drawer {} name appears incorrect".format(i))
+                if fan_drawer_truth:
+                    self.expect(name == fan_drawer_truth[i]['name'], "Fan_drawer {} name does not match".format(i))
```

Comment:
Can we include the expected name as well? It might be more clear where the mismatch is if we can compare expected v. received.

Reply:
fixed